network/patton/smartnode/snmp/mode/call.pm

```diff
@@ -0,0 +1,185 @@
+#
+# Copyright 2022 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package network::patton::smartnode::snmp::mode::call;
+
+use base qw(centreon::plugins::templates::counter);
+
+use strict;
+use warnings;
+use Digest::MD5 qw(md5_hex);
+
+sub prefix_gateway_output {
+    my ($self, %options) = @_;
+    return "Gateway '".$options{instance_value}->{display}."' ";
+}
+
+sub prefix_isdn_output {
+    my ($self, %options) = @_;
+    return "ISDN '".$options{instance_value}->{display}."' ";
+}
+
+sub set_counters {
+    my ($self, %options) = @_;
+
+     $self->{maps_counters_type} = [
+        { name => 'gateway', type => 1, cb_prefix_output => 'prefix_gateway_output', message_multiple => 'All gateways are ok' },
+        { name => 'isdn', type => 1, cb_prefix_output => 'prefix_isdn_output', message_multiple => 'All ISDN are ok' }
+    ];
+
+    $self->{maps_counters}->{gateway} = [
+        { label => 'gateway-calls-connected-count', nlabel => 'gateway.calls.connected.count', set => {
+                key_values => [ { name => 'gateway_calls_connected_count' }, { name => 'display' } ],
+                output_template => 'connected calls: %d',
+                perfdatas => [
+                    { template => '%d', min => 0, unit => '', label_extra_instance => 1 }
+                ]
+            }
+        },
+        { label => 'gateway-calls-ongoing-count', nlabel => 'gateway.calls.ongoing.count', set => {
+                key_values => [ { name => 'gateway_calls_ongoing_count' }, { name => 'display' } ],
+                output_template => 'ongoing calls: %d',
+                perfdatas => [
+                    { template => '%d', min => 0, unit => '', label_extra_instance => 1 }
+                ]
+            }
+        },
+        { label => 'gateway-calls-accumulated-count', nlabel => 'gateway.calls.accumulated.count', set => {
```

Comment:
please simplify the label: it's not the same than the perfdata label. remove the count